_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -0,0 +1,599 @@
+									/* translators: %s: plugin name and version */
+									esc_attr( sprintf( __( 'Install %s now', 'jetpack' ), $name ) ),
+									esc_attr( $name ),
+									esc_attr( $name ),
+									__( 'Install Now', 'jetpack' )
+								);
+							} else {
+								$action_links[] = sprintf(
+									'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
+									_x( 'Cannot Install', 'plugin', 'jetpack' )
+								);
+							}
+						}
+						break;
+
+					case 'update_available':
+						if ( $status['url'] ) {
+							$action_links[] = sprintf(
+								'<a class="update-now button aria-button-if-js jptracks" data-plugin="%s" data-slug="%s" href="%s" aria-label="%s" data-name="%s" data-jptracks-name="jetpack_about_update_button" data-jptracks-prop="%s">%s</a>',
+								esc_attr( $status['file'] ),
+								esc_attr( $plugin['slug'] ),
+								esc_url( $status['url'] ),
+								/* translators: %s: plugin name and version */
+								esc_attr( sprintf( __( 'Update %s now', 'jetpack' ), $name ) ),
+								esc_attr( $name ),
+								esc_attr( $name ),
+								__( 'Update Now', 'jetpack' )
+							);
+						}
+						break;
+
+					case 'latest_installed':
+					case 'newer_installed':
+						if ( is_plugin_active( $status['file'] ) ) {
+							$action_links[] = sprintf(
+								'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
+								_x( 'Active', 'plugin', 'jetpack' )
+							);
+						} elseif ( current_user_can( 'activate_plugin', $status['file'] ) ) {
+							$button_text = __( 'Activate', 'jetpack' );
+							/* translators: %s: plugin name */
+							$button_label = _x( 'Activate %s', 'plugin', 'jetpack' );
+							$activate_url = add_query_arg(
+								array(
+									'_wpnonce' => wp_create_nonce( 'activate-plugin_' . $status['file'] ),
+									'action'   => 'activate',
+									'plugin'   => $status['file'],
+								),
+								network_admin_url( 'plugins.php' )
+							);
+
+							if ( is_network_admin() ) {
+								$button_text = __( 'Network Activate', 'jetpack' );
+								/* translators: %s: plugin name */
+								$button_label = _x( 'Network Activate %s', 'plugin', 'jetpack' );
+								$activate_url = add_query_arg( array( 'networkwide' => 1 ), $activate_url );
+							}
+
+							$action_links[] = sprintf(
+								'<a href="%1$s" class="button activate-now" aria-label="%2$s" data-jptracks-name="jetpack_about_activate_button" data-jptracks-prop="%3$s">%4$s</a>',
+								esc_url( $activate_url ),
+								esc_attr( sprintf( $button_label, $plugin['name'] ) ),
+								esc_attr( $plugin['name'] ),
+								$button_text
+							);
+						} else {
+							$action_links[] = sprintf(
+								'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
+								_x( 'Installed', 'plugin', 'jetpack' )
+							);
+						}
+						break;
+				}
+			}
+
+			$details_link = self_admin_url(
```

Comment:
This does not quite work on Multisite installs; if you click on the "More Details" link, the modal pops up but you won't see the option to install that plugin; you are in fact redirected to `wp-admin/network/plugin-install.php` with no parameters inside that modal.

Reply:
Leaving to @crunnells so he decides what should we do.

Reply:
@eliorivero @jeherve Is there a way we can link Multisite to the plugin install page with the slug pre-populated in the search field? I copied the code that drives the plugin page here, but didn't think multisite would be different. I'd rather not spend a bunch of time re-inventing the wheel here.

Reply:
Let me know what you gents can figure out here. I don't personally see this as a blocker. Our multisite installs are fairly low, aren't they? If all else fails we can just send all multisite installs to the plugins page in wp-admin when any of the links in the about page are clicked.

Reply:
Not sure if this helps with context at all, but I basically copied the `[WP_Plugin_Install_List_Table::display_rows()](https://core.trac.wordpress.org/browser/tags/5.1.1/src/wp-admin/includes/class-wp-plugin-install-list-table.php#L437)` method to display the plugin info here.

Reply:
> Is there a way we can link Multisite to the plugin install page with the slug pre-populated in the search field?

On multisite, the plugin search URL looks like this:

`https://mainnetworkurl.com/wp-admin/network/plugin-install.php?s=wp-super-cache&tab=search&type=term`

Reply:
Fixed